doc/run_with_maxcompute.md

```diff
@@ -0,0 +1,30 @@
+# How to connect Maxcompute with SQLFlow
+
+This tutorial explains how to connect SQLFlow with [Maxcompute (a.k.a ODPS)](https://www.alibabacloud.com/product/maxcompute).
+
+## Connect Existing Maxcompute server
+
+To connect an existed Maxcompute server instance, we need to configure a `datasource` string in the format of   
+`maxcompute://{accesskey_id}:{accesskey_secret}@{endpoint}?current_project={curr_project}&scheme={scheme}`
+
+In the above format,
+- `accesskey_id:accesskey_secret` are the API keys for you to access aliyun. You may find it at the [user center of aliyun](https://usercenter.console.aliyun.com/#/manage/ak) after login.
+- `endpoint`. You may find it through the [workbench of aliyun](https://workbench.data.aliyun.com/console?#/) and the [configure endpoints](https://www.alibabacloud.com/help/doc-detail/34951.htm). In the workbench page, let's find the region in the workspace block, e.g.`China North 2`. Then, in the configure endpoints page, we may find out the public endpoint corresponding to the region, in this case, is service.cn-beijing.maxcompute.aliyun-inc.com/api. 
+   
+   **Note**: actually the whole endpoint is http://service.cn-beijing.maxcompute.aliyun-inc.com/api. We just take service.cn-beijing.maxcompute.aliyun-inc.com/api as the endpoint and the protocol(http) as `schema_name`.
```

Comment:
Maybe service.cn-beijing.maxcompute.aliyun-inc.com/api => `service.cn-beijing.maxcompute.aliyun-inc.com/api`.

Reply:
"actually the whole ..." => "please be aware that the whole ...".